org.ekstep.sunbirdcommonheader-1.5/editor/headerApp.js

```diff
@@ -59,6 +58,56 @@ angular.module('org.ekstep.sunbirdcommonheader:app', ["Scope.safeApply", "yaru22
     $scope.isFlagReviewer = false;
     $scope.editorEnv = "";
     $scope.showEditMeta = true;
+    $scope.contentCredits = [];
+    $scope.listLimit = 5;
+    /*
+    * Add owner details and update current count with new values.
+    */
+    $scope.updateOwnershipList = function(event, nodeData) {
+        var contentCredit = {'ownedBy':nodeData.data.metadata.identifier, 
```

Comment:
Add one more key "resourcesAdded": to this object. Use this key while removing/deleting the resource. 
If resourcesAdded = 1, then you can remove the user from the list directly.
If resourcesAdded = 2, then you can should remove the user from the list, you just have to decrement the resourceAdded count.

Reply:
This logic will help only if removing single resource at a time but removing a child and that child have another child then this logic will not help.